templates/rails_helper.rb

```diff
@@ -14,8 +14,9 @@ module Features
 
 RSpec.configure do |config|
   config.before(:each, js: true) do
-    page.driver.block_unknown_urls
+    page.block_unknown_urls = true
```

Comment:
I'm surprised this works. The `page` object here is a `Capybara::Session`, and I don't know how this property would be defined on it.